ckanext/saml2auth/templates/user/snippets/login_form.html

```diff
@@ -18,5 +18,5 @@
 {% ckan_extends %}
 {% block login_button %}
   <button class="btn btn-primary" type="submit">{{ _('Login') }}</button>
-  <a class="btn btn-default" href="{{ h.url_for('saml2auth.saml2login') }}">{{ _('SSO') }}</a>
+  <a class="btn btn-default" href="{{ h.url_for('saml2auth.saml2login') }}">{{ _('Login with') }} {{ (h.get_saml2auth_login_button_text()) }}</a>
```

Comment:
```suggestion
  <a class="btn btn-default" href="{{ h.url_for('saml2auth.saml2login') }}">{{ (h.get_saml2auth_login_button_text()) }}</a>
```
Let user define the full text